_data/security.yml

```diff
@@ -60,6 +60,7 @@ websites:
     img: cloudflare.png
     tfa:
       - totp
+      - hardware
```

Comment:
```suggestion
      - u2f
```

Reply:
Is there any rule for using u2f vs hardware? Let's take GitHub as example, where 'hardware' is listed instead of 'u2f'.

Reply:
If they support U2F keys, add the u2f tag. If they support something else, add the hardware tag.

We rely on user contributions so some sites might not be up to date.

Reply:
Got it, thanks. I'll take a look at the sites with the "hardware" tag later to see if any of them needs to be updated.